packages/block-library/src/column/edit.js

```diff
@@ -51,7 +51,63 @@ function ColumnEdit( {
 		} );
 	};
 
-	const hasWidth = Number.isFinite( width );
+	const hasWidth = ( isNaN( width ) && width ) || Number.isFinite( width );
+
+	const isWidthValid = () => {
+		const validUnits = [
+			'fr',
+			'rem',
+			'em',
+			'ex',
+			'%',
+			'px',
+			'cm',
+			'mm',
+			'in',
+			'pt',
+			'pc',
+			'ch',
+			'vh',
+			'vw',
+			'vmin',
+			'vmax',
+		];
+
+		// Return true for values that can be passed-on as-is.
+		if (
```

Comment:
These two cases are already covered by `! width`
```
'' === width ||
0 === width
```

In general this could be more concise with something like this:
```
if ( ! width || [ '0', 'auto', 'inherit', 'initial' ].includes( width ) ) return true
```

Reply:
Thanks, fixed :+1: